frontend/src/loaders/submission-overview-loader.ts

```diff
@@ -27,10 +27,16 @@ export default async function loadSubmissionOverview({
 }: {
   params: Params<string>;
 }) {
+
   const projectId = params.projectId;
   const projectResponse = await authenticatedFetch(
     `${APIURL}/projects/${projectId}`
   );
+
+  if(projectResponse.status != 200) {
+    throw new Response("Not authenticated", {status: 403});

```

Comment:
You can simply return the response, this is very unnecessary

Reply:
No it doesn't work then.

Reply:
Then return a HTTP response